DCS-SR-Client/UI/ClientWindow/InputBindingControl.xaml.cs

```diff
@@ -35,7 +35,7 @@ public InputDeviceManager InputDeviceManager
         public void LoadInputSettings()
         {
             DeviceLabel.Content = InputName;
-            ModifierLabel.Content = InputName + " Modifier";
```

Comment:
not sure i follow what is going on in this line

Reply:
![image](https://github.com/FPGSchiba/VNGD-SimpleRadioStandalone/assets/119721976/166949d5-696b-452e-8c96-a7e6579233c5)

I removed the input name being included before the name modifier and put in a lot of spaces to "tab" modifier over

Reply:
Originally it said

__________________________
Radio 2
Radio 2 Modifier
---------------------------

Reply:
@gtodd876

Reply:
is spaces the best way to achieve layout in this situation?

Reply:
It was cheap and very easy to accomplish. Additionally, it's very easy to revert back to something else without needing to change the main XAML formatting. We can always move the different cells around in XAML, but this was pretty easy and involved merely seconds to create.

Reply:
sounds good